### DISCUSSION TITLE
URI scheme - Outlook

### DISCUSSION THREAD

**Discussion by torbenriis** [MicrosoftDocs/office-developer-client-docs#668] [Q&A]
Are there any plans for supporting URI scheme integration to Outlook? Fx. for some of the arguments that already exists for outlook.exe and with the possibility to specify a http resource (webdav) containing the msg file. Just as it is possible with ms-word, ms-excel, etc.

We are particular interested in the integration where we provide the http resource to the msg file externally. We are an enterprise archive/document management system, so the emails isn’t kept anymore on the exchange server. It would be fantastic to be able to reply or forward an email using standard Office URI schemes. Today we solve it by registering a custom protocol handler that opens outlook.exe with the desired arguments. But being able to do it without a custom protocol handler, just as for other office documents, would be great.

Kind regards
Torben Riis

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cc9d3afc-5f18-8c4f-1b9c-fd2fe52f49db
* Version Independent ID: a900d01a-0f42-13b8-3289-a2787f7b8499
* Content: [Office URI Schemes](https://docs.microsoft.com/en-us/office/client-developer/office-uri-schemes)
* Content Source: [docs/office-uri-schemes.md](https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/office-uri-schemes.md)
* Product: **office365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by lindalu-MSFT** [comment]
Hi @torbenriis, last time I asked, URI support was on the backlog. I will try to follow up with our engineers and see what the status for Office URI scheme integration is. I know this is a big request, so you can also go to [Outlook UserVoice](https://outlook.uservoice.com/) and upvote the request.

**Comment by torbenriis** [comment]
Hi @lindalu-MSFT 

I searched a little bit around on Outlook UserVoice, but I did actually only find two issues that was around Office URI scheme integration, where one was about integration from IOS and the other about integration from Office 365 Online. Both were regarding needed URI scheme integration for Outlook, but not for the desktop version. Should I create a new suggestion or were I just not able to find the one you were thinking on?

The integration I’m thinking on is the one that already exists for word, excel, etc. but lacking support for outlook.
https://docs.microsoft.com/en-us/office/client-developer/office-uri-schemes

**Comment by torbenriis** [comment]
Anything new on this?
